recipes-core/systemd/systemd_%.bbappend

```diff
@@ -90,18 +88,38 @@ do_install:append:phyboard-polis-imx8mm-4() {
     enable_hardware_watchdog
 }
 
-# adapt tauri-l systemd-networkd-wait-online.service state
-do_install:append:phygate-tauri-l-imx8mm-2() {
-    sed -i -e 's#^ExecStart=\(.*\)#ExecStart=/bin/bash -c \x27\1 --any --interface=${OMNECT_ETH0} --interface=${OMNECT_ETH1} --timeout=\${OMNECT_WAIT_ONLINE_TIMEOUT_IN_SECS:-300}\x27#' \
+
+def online_ifc_list_to_parameter_list(d, ifclistvar):
+    param_list = ''
+    ifclist = d.getVar(ifclistvar)
+    if ifclist == None:
+        bb.warn('No online interfaces defined in variable {}!'.format(ifclistvar))
+        return param_list
+    interfaces = ifclist.split(':')
+    if len(interfaces) > 1:
+        param_list = '--any '
+    for i in interfaces:
+        param_list += '--interface={} '.format(i)
+    return param_list
+
+ONLINE_INTERFACE_ARGS = "${@online_ifc_list_to_parameter_list(d, 'OMNECT_ONLINE_INTERFACES')}"
+
+do_install:append() {
+    sed -i -e 's#^ExecStart=\(.*\)#ExecStart=/bin/bash -c \x27\1 \${OMNECT_WAIT_ONLINE_INTERFACES:-${ONLINE_INTERFACE_ARGS}} --timeout=\${OMNECT_WAIT_ONLINE_TIMEOUT_IN_SECS:-300}\x27#' \
         ${D}${systemd_system_unitdir}/systemd-networkd-wait-online.service
+    if ${@bb.utils.contains('DISTRO_FEATURES', '3g', 'true', 'false', d)}; then
+	install -d ${D}${sysconfdir}/systemd/system/aziot-identityd.service.d
```

Comment:
we shouldn't use `sysconfdir`  ... that's from a "systemd admin" point of view for the customer. 
we shouldn't install overrides ourselves.

Reply:
now only aziot-identityd.inc is used.